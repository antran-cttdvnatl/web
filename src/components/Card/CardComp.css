.introduction .card {
    border: none;
}

.introduction {
    font-family: 'Lato', sans-serif;
    font-size: 20px;
}

.introduction h5{
    font-size: 25px;
}

.introduction h3, .introduction h5, .introduction .h5 {
    font-weight: 600;
}

.introduction h3 {
    text-align: center;
}
.introduction .card-text {
    font-size: 20px;
}

.introduction .card-deck {
    width: 1200px;
}

.introduction .content {
    justify-content: center;
}

.introduction .card-body {
    padding: 0;
}
