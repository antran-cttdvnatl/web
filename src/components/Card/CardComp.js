import React from "react";
import CardDeck from 'react-bootstrap/CardDeck';
import Card from 'react-bootstrap/Card';
import './style.css';

const CardComp = () => (

    <div className="card-news">
        <CardDeck>
            <Card>
                <Card.Body>
                    <Card.Title>Title</Card.Title>
                    <Card.Text>
                        write what you want here
                    </Card.Text>
                </Card.Body>
                <Card.Footer>
                    <small>Footer text</small>
                </Card.Footer>
            </Card>
            <Card>
                <Card.Body>
                    <Card.Title>Title</Card.Title>
                    <Card.Text>
                        write what you want here
                    </Card.Text>
                </Card.Body>
                <Card.Footer>
                    <small>Footer text</small>
                </Card.Footer>
            </Card>
            <Card>
                <Card.Body>
                    <Card.Title>Title</Card.Title>
                    <Card.Text>
                        write what you want here
                    </Card.Text>
                </Card.Body>
                <Card.Footer>
                    <small>Footer text</small>
                </Card.Footer>
            </Card>
        </CardDeck>;
    </div>

)

export default CardComp;