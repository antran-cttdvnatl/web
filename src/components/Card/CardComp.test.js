import React from 'react';
import renderer from 'react-test-renderer';
import CardComp from './CardComp';


it('Card should render correctly', () => {
    const component = renderer.create(
        <CardComp />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})