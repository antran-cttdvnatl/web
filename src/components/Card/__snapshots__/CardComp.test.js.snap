// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render correctly 1`] = `
<section
  className="introduction"
>
  <div
    className="header"
  >
    <h3>
      Welcome To CTTDVNATL
    </h3>
  </div>
  <div
    className="row content"
  >
    <div
      className="card-deck"
    >
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          History
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Mission
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Construction
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`CardComp should render correctly 1`] = `
<section
  className="introduction"
>
  <div
    className="header"
  >
    <h3>
      Welcome To CTTDVNATL
    </h3>
  </div>
  <div
    className="row content"
  >
    <div
      className="card-deck"
    >
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          History
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Mission
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Construction
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Main Menu should render correctly 1`] = `
<section
  className="introduction"
>
  <div
    className="header"
  >
    <h3>
      Welcome To CTTDVNATL
    </h3>
  </div>
  <div
    className="row content"
  >
    <div
      className="card-deck"
    >
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          History
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Mission
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <a
          href="/home"
        >
          <img
            className="card-img"
            src="./images/internal.jpg"
          />
        </a>
        <div
          className="card-title h5"
        >
          Construction
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Started at the end of 2018 and finish at the end of 2019.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
